AnnoyancesFilter/sections/annoyances.txt

```diff
@@ -1484,6 +1484,11 @@ oneplus.com,tickets.ua,woman.ru,elle.ru,marieclaire.ru,europaplus.ru,krossovki.c
 beautyinsider.ru###articleSubscribeBlock
 officelife.media##.subscribe-plate
 pinsknews.by#?#.sidebar_list > li.widget:has(> div.textwidget > form[action] strong:contains(Подпишитесь))
+politico.com#?#.below-article-section div.below-story-text:has(> aside > section > div.sign-up__column > div.sign-up__summary > a[href="https://www.politico.com/newsletters"])
```

Comment:
Not `.below-article-section div.below-story-text > aside:has(`? Current rule blocks too much.
https://www.politico.com/news/2020/08/07/rashida-tlaib-campaign-finance-violation-392579

Reply:
You are right